### PR TITLE
Feedback from Support re support topic and KB link

### PR DESCRIPTION
**Pull request by drothery-edb**
## What Changed?

Changed support topic title and KB link label

### CODE REVIEW

**Comment by josh-heyer** [comment]
I've changed the other place this string appears as well, @drothery-edb 

But... I have a bone to pick with the use of `/` here - its meaning is that of the inclusive "and" (knowledge base AND technical alerts) but that meaning is somewhat ambiguous (could also mean "or") and we don't seem to be using this anywhere else in the navigation; I would recommend either just SAYING "and" or using an ampersand here.

**Review by josh-heyer — APPROVED** [review]

File: src/components/index-sub-nav.js [hunk 1]
```diff
@@ -14,7 +14,7 @@ const IndexSubNav = () => (
     <ul className="list-inline mr-3">
       <IndexSubLink url="https://www.enterprisedb.com/">EDB Home</IndexSubLink>
       <IndexSubLink url="https://support.enterprisedb.com/support/s/">
-        Knowledge Base
+        Knowledge Base / Technical Alerts
```
Comment (drothery-edb) [inline src/components/index-sub-nav.js:17]:
```suggestion
        Knowledge Base and Technical Alerts
```
```suggestion
        Knowledge Base / Technical Alerts
```

File: src/components/side-navigation.js [hunk 2]
```diff
@@ -23,7 +23,7 @@ const SideNavigationFooter = () => (
   <ul className="list-unstyled mt-0">
     <hr />
     <FooterItem url="https://support.enterprisedb.com/support/s/">
-      Knowledge Base
+      Knowledge Base / Technical Alerts
```
Comment (drothery-edb) [inline src/components/side-navigation.js:26]:
```suggestion
      Knowledge Base and Technical Alerts
```
```suggestion
      Knowledge Base / Technical Alerts
```

**Review by josh-heyer — APPROVED** [review]
👍👍👍

**Pull request merged**